### DISCUSSION TITLE
Плохо грузит плеер Ютуба

### DISCUSSION THREAD

**Discussion by jAba65** [Anonym-tsk/nfqws-keenetic#89] [Обсуждения]
**Опишите проблему**
обновил nfqws до 09.10, стал плохо грузится плеер ютуба, бывает по 2-3 минуты

**Модель маршрутизатора**
Keenetic Giga (KN-1011)

**Провайдер**
Ростелеком/ppp0

**Выполните команды и приложите их вывод**
`opkg info nfqws-keenetic`
```
Package: nfqws-keenetic
Version: 2.4.1
Depends: iptables, busybox
Conflicts: tpws-keenetic
Status: install user installed
Section: net
Architecture: all
Size: 726199
Filename: nfqws-keenetic_2.4.1_all_entware.ipk
Conffiles:
 /opt/etc/nfqws/nfqws.conf b49e99e9938987df7d24e87d8b87f69cf1f39c3ddf0368462c92bf36e8bbe3a5
 /opt/etc/nfqws/user.list 45dc2adaa172b86d73369c6ed12a8a0e648b851b66293b11514c3b1d4bd3fce6
 /opt/etc/nfqws/auto.list e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
 /opt/etc/nfqws/exclude.list 31fbbe06a06e48f9047db40300750b7f11c203a7434fb1a3151f33518ccd9805
Description: NFQWS service
Installed-Time: 1728501543
```

`/opt/etc/init.d/S51nfqws restart`
```
Stopping NFQWS service...
we have 3 user defined desync profile(s) and default low priority profile 0
Loading hostlist /opt/etc/nfqws/auto.list
loading plain text list
Loaded 18 hosts from /opt/etc/nfqws/auto.list
Loading hostlist /opt/etc/nfqws/user.list
loading plain text list
Loaded 91320 hosts from /opt/etc/nfqws/user.list
Loading hostlist /opt/etc/nfqws/auto.list
loading plain text list
Loaded 18 hosts from /opt/etc/nfqws/auto.list
Loading hostlist /opt/etc/nfqws/user.list
loading plain text list
Loaded 91320 hosts from /opt/etc/nfqws/user.list
Loading hostlist /opt/etc/nfqws/exclude.list
loading plain text list
Loaded 26 hosts from /opt/etc/nfqws/exclude.list
Loading hostlist /opt/etc/nfqws/exclude.list
loading plain text list
Loaded 26 hosts from /opt/etc/nfqws/exclude.list
Started NFQWS service
```

`cat /opt/etc/nfqws/nfqws.conf`
```
# Provider network interface, e.g. eth3
# You can specify multiple interfaces separated by space, e.g. ISP_INTERFACE="ppp0"
ISP_INTERFACE="ppp0"

# All arguments here: https://github.com/bol-van/zapret (search for `nfqws` on the page)
# HTTP(S) strategy
NFQWS_ARGS="--filter-udp=50000-65535 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-cutoff=d4 --dpi-desync-repeats=6 --new --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-split-pos=1 --dpi-desync-fooling=md5sig,badseq --dpi-desync-fake-tls=/opt/etc/nfqws/tls_clienthello.bin"

# QUIC strategy
NFQWS_ARGS_QUIC="--dpi-desync=fake --dpi-desync-repeats=6 --dpi-desync-cutoff=d4 --dpi-desync-fooling=badsum --dpi-desync-fake-quic=/opt/etc/nfqws/quic_initial.bin"

# auto - automatically detects blocked resources and adds them to the auto.list
NFQWS_EXTRA_ARGS="--hostlist=/opt/etc/nfqws/user.list --hostlist-auto=/opt/etc/nfqws/auto.list --hostlist-auto-debug=/opt/var/log/nfqws.log --hostlist-exclude=/opt/etc/nfqws/exclude.list"

# list - applies rules only to domains in the user.list
#NFQWS_EXTRA_ARGS="--hostlist=/opt/etc/nfqws/user.list"

# all  - applies rules to all traffic except domains from exclude.list
#NFQWS_EXTRA_ARGS="--hostlist-exclude=/opt/etc/nfqws/exclude.list"

# IPv6 support
IPV6_ENABLED=0

# HTTP support
HTTP_ENABLED=0

# QUIC support
QUIC_ENABLED=1
UDP_PORTS=443,50000:65535

# Syslog logging level (0 - silent, 1 - debug)
LOG_LEVEL=0

NFQUEUE_NUM=200
USER=nobody
CONFIG_VERSION=3
```

`ps | grep nfqws`
```
12141 nobody   16124 S    /opt/usr/bin/nfqws --daemon --pidfile=/opt/var/run/nfqws.pid --user=nobody --qnum=200 --fil
12185 root      5400 S    grep nfqws
```

`iptables-save | grep 200`
```
-A POSTROUTING -o ppp0 -p udp -m multiport --dports 443,50000:65535 -m connbytes --connbytes 1:8 --connbytes-mode packets --connbytes-dir original -m mark ! --mark 0x40000000/0x40000000 -j NFQUEUE --queue-num 200 --queue-bypass
-A POSTROUTING -o ppp0 -p tcp -m tcp --dport 443 -m connbytes --connbytes 1:8 --connbytes-mode packets --connbytes-dir original -m mark ! --mark 0x40000000/0x40000000 -j NFQUEUE --queue-num 200 --queue-bypass
```

`sysctl net.netfilter.nf_conntrack_checksum`
```
net.netfilter.nf_conntrack_checksum = 0
```

**Comment by jAba65** [comment]
Так же перестал работать Ютуб на телефоне

**Comment by Riadner** [comment]
Почему параметр --dpi-desync= повторяется 2 раза в одной строке в конфигурации? Это нормально? На Дом.ру тоже отвалился в последний час.

**Comment by jAba65** [comment]
это разные стратегии, конфиг дефолтный

**Comment by Riadner** [comment]
![Снимок экрана 2024-10-09 224927](https://github.com/user-attachments/assets/3689376a-9236-41c2-9616-b11e749f339a)

**Comment by jAba65** [comment]
> ![Снимок экрана 2024-10-09 224927](https://private-user-images.githubusercontent.com/2353070/375110920-3689376a-9236-41c2-9616-b11e749f339a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1MDM3ODIsIm5iZiI6MTcyODUwMzQ4MiwicGF0aCI6Ii8yMzUzMDcwLzM3NTExMDkyMC0zNjg5Mzc2YS05MjM2LTQxYzItOTYxNi1iMTFlNzQ5ZjMzOWEuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDlUMTk1MTIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmVhMWJkYTQwMzQ3NTY2YWFiN2MyNzMzYmM0NGI5NzFiYzU0NDcyNTMxODNlZmNiNDY4MGYxOWJmNWZmYTcxNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.gSDXfaPhbOIDeNBGP5doxuXXHDgNrvkXdxLu1Uwt0fU)

тоже самое

**Comment by DDefou** [comment]
аналогично, но обновил пакет и все работает как прежде

**Comment by HerosteveFeed** [comment]
Такая же проблема, несколько часов назад всё отлично работало

**Comment by gudvindll** [comment]
пакет обновлял что бы попасть в ds, Дискорд заработал, отвал ютуба случился.

**Comment by SVY76** [comment]
Отвалился ютуб повсюду

**Comment by elijah0107** [comment]
Тоже перестал грузить

**Comment by ops244** [comment]
а вы что думали это панацея? да сюда ссылка на каждом третьем форуме, естественно все заблочат xD

**Comment by gudvindll** [comment]
Половину ночи пробовал всемозможные стратегии, Discord работает как до блокировки, когда дело доходит до Youtube все летит к чертям. Либо через 2-3 минуты начинает играть ролик. либо цикличная загрузка. На девайсах типа телефона или телевизора вообще эффект нулевой. Миниатюры грузятся, при попытке перейти в видео бесконечная загрузка, даже комментарии не грузятся. Пробовал откатиться на 2.3.3 . результат такой же с youtube. Хотя вчера до обновления все работало прекрасно

**Comment by HerosteveFeed** [comment]
Помогло изменение значения --dpi-desync-repeats на 20, ютуб и инстаграм вновь стали работать

**Comment by gudvindll** [comment]
> Помогло изменение значения --dpi-desync-repeats на 20, ютуб и инстаграм вновь стали работать

А конфиг не покажите? Я откатился на 2.3.3 с костылями ютуб пашет, но буферизация долгая

Был бы рад починить все на новой версии

**Comment by kolek15** [comment]
> Я откатился на 2.3.3 с костылями ютуб пашет, но буферизация долгая
А как откатиться? какую команду прописать?

**Comment by HerosteveFeed** [comment]
> > Помогло изменение значения --dpi-desync-repeats на 20, ютуб и инстаграм вновь стали работать
> 
> А конфиг не покажите? Я откатился на 2.3.3 с костылями ютуб пашет, но буферизация долгая
> 
> Был бы рад починить все на новой версии


```
NFQWS_ARGS="--filter-udp=50000-65535 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-cutoff=d4 --dpi-desync-repeats=20 --new --dpi-desync=fake,disorder2 --dpi-desync-ttl=3 --dpi-desync-split-pos=1 --dpi-desync-repeats=20 --dpi-desync-fooling=md5sig,badseq --dpi-desync-fake-tls=/opt/etc/nfqws/tls_clienthello.bin"

NFQWS_ARGS_QUIC="--dpi-desync=fake --dpi-desync-repeats=20 --dpi-desync-cutoff=d4 --dpi-desync-fooling=badsum --dpi-desync-fake-quic=/opt/etc/nfqws/quic_initial.bin"

UDP_PORTS=443,444,50000:65535
```
Я не уверен, насколько данные настройки не являются костыльными, но как говорится, работает и слава богу. Оператор Билайн.

**Comment by gudvindll** [comment]
> > Я откатился на 2.3.3 с костылями ютуб пашет, но буферизация долгая
> > А как откатиться? какую команду прописать?
а там достаточно просто все. сначала бекапы конфигов советую сделать

opkg remove nfqws-keenetic
opkg update
wget https://anonym-tsk.github.io/nfqws-keenetic/all/nfqws-keenetic_2.3.3_all_entware.ipk
opkg install nfqws-keenetic_2.3.3_all_entware.ipk

**Comment by kolek15** [comment]
> > > Я откатился на 2.3.3 с костылями ютуб пашет, но буферизация долгая
> > > А как откатиться? какую команду прописать?
> > > а там достаточно просто все. сначала бекапы конфигов советую сделать
> 
> opkg remove nfqws-keenetic opkg update wget https://anonym-tsk.github.io/nfqws-keenetic/all/nfqws-keenetic_2.3.3_all_entware.ipk opkg install nfqws-keenetic_2.3.3_all_entware.ipk

Откатился, но yt так и не грузит

**Comment by gudvindll** [comment]
> > > > Я откатился на 2.3.3 с костылями ютуб пашет, но буферизация долгая
> > > > А как откатиться? какую команду прописать?
> > > > а там достаточно просто все. сначала бекапы конфигов советую сделать
> > 
> > 
> > opkg remove nfqws-keenetic opkg update wget https://anonym-tsk.github.io/nfqws-keenetic/all/nfqws-keenetic_2.3.3_all_entware.ipk opkg install nfqws-keenetic_2.3.3_all_entware.ipk
> 
> Откатился, но yt так и не грузит

В файле user.list что?

у меня там 
cat /opt/etc/nfqws/user.list
```
googlevideo.com
youtubei.googleapis.com
i.ytimg.com

discord-attachments-uploads-prd.storage.googleapis.com
dis.gd
discord.co
discord.com
discord.design
discord.dev
discord.gg
discord.gift
discord.gifts
discord.media
discord.new
discord.store
discord.tools
discordapp.com
discordapp.net
discordmerch.com
discordpartygames.com
discord-activities.com
discordactivities.com
discordsays.com
discordstatus.com
discord.com
discord.gg
discordapp.com
discord-attachments-uploads-prd.storage.googleapis.com
dis.gd
discordstatus.com
discord.media
discordapp.net
discordcdn.com
discord.dev
discord.new
discord.gift
discord.co

rutracker.org
```

**Comment by x12lab** [comment]
> В файле user.list что?

Лично у меня используется exclude mode (всё, кроме хостов из списка), поэтому user.list не играет роли.

На проводном билайне не работает ни одна из прошлых стратегий, а если сделать --dpi-desync-repeats=20, то иногда в yt что-то прорывается, но перед этим ооооочень сильно и долго тупит. В основном проигрыватель просто висит и ничего не грузит.

Все остальное кроме yt работает.

**Comment by gudvindll** [comment]
> > > Помогло изменение значения --dpi-desync-repeats на 20, ютуб и инстаграм вновь стали работать
> 
> > 
> 
> > А конфиг не покажите? Я откатился на 2.3.3 с костылями ютуб пашет, но буферизация долгая
> 
> > 
> 
> > Был бы рад починить все на новой версии
> 
> 
> 
> 
> 
> ```
> 
> NFQWS_ARGS="--filter-udp=50000-65535 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-cutoff=d4 --dpi-desync-repeats=20 --new --dpi-desync=fake,disorder2 --dpi-desync-ttl=3 --dpi-desync-split-pos=1 --dpi-desync-repeats=20 --dpi-desync-fooling=md5sig,badseq --dpi-desync-fake-tls=/opt/etc/nfqws/tls_clienthello.bin"
> 
> 
> 
> NFQWS_ARGS_QUIC="--dpi-desync=fake --dpi-desync-repeats=20 --dpi-desync-cutoff=d4 --dpi-desync-fooling=badsum --dpi-desync-fake-quic=/opt/etc/nfqws/quic_initial.bin"
> 
> 
> 
> UDP_PORTS=443,444,50000:65535
> 
> ```
> 
> Я не уверен, насколько данные настройки не являются костыльными, но как говорится, работает и слава богу. Оператор Билайн.

Сейчас обновился для теста на 2.4.1 сохранил все конфиги, применил рекомендации комрада, ютуб заработал везде и дискорд тоже.
User.list использовал свой и коментов выше. Пока полет нормальный. Дети в восторге, жена в инсте опять залипает.

Подкинули проблем на нашу голову конечно.

**Reply by Flatty11** [reply]
> в инсте

Подкиньте пж список хостов, чтобы инста работала)

**Comment by kolek15** [comment]
> Сейчас обновился для теста на 2.4.1 сохранил все конфиги, применил рекомендации комрада, ютуб заработал везде и дискорд тоже. User.list использовал свой и коментов выше. Пока полет нормальный. Дети в восторге, жена в инсте опять залипает.
> 
> Подкинули проблем на нашу голову конечно.

Тоже самое сделал, заработало:
nfqws.conf:
`# HTTP(S) strategy
NFQWS_ARGS="--filter-udp=50000-65535 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-cutoff=d4 --dpi-desync-repeats=20 --new --dpi-desync=fake,disorder2 --dpi-desync-ttl=3 --dpi-desync-split-pos=1 --dpi-desync-repeats=20 --dpi-desync-fooling=md5sig,badseq --dpi-desync-fake-tls=/opt/etc/nfqws/tls_clienthello.bin"`

`# QUIC strategy
NFQWS_ARGS_QUIC="--dpi-desync=fake --dpi-desync-repeats=20 --dpi-desync-cutoff=d4 --dpi-desync-fooling=badsum --dpi-desync-fake-quic=/opt/etc/nfqws/quic_initial.bin"`

user.list:
```
youtube.com
youtu.be
googleapis.com
googlevideo.com
i.ytimg.com
i9.ytimg.com
yt3.ggpht.com
yt3.googleusercontent.com
yt4.ggpht.com
yt4.googleusercontent.com
gvt1.com
gstatic.com
youtubei.googleapis.com
youtube-ui.l.google.com
ytimg.l.google.com
ytstatic.l.google.com
discord-attachments-uploads-prd.storage.googleapis.com
dis.gd
discord.co
discord.com
discord.design
discord.dev
discord.gg
discord.gift
discord.gifts
discord.media
discord.new
discord.store
discord.tools
discordapp.com
discordapp.net
discordmerch.com
discordpartygames.com
discord-activities.com
discordactivities.com
discordsays.com
discordstatus.com
discordcdn.com
rutracker.org

```

**Reply by kolek15** [reply]
v2.4.2 yt с этими настройками работает. На стоковых настройках не работает.

**Reply by Evesh** [reply]
> `# HTTP(S) strategy NFQWS_ARGS="--filter-udp=50000-65535 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-cutoff=d4 --dpi-desync-repeats=20 --new --dpi-desync=fake,disorder2 --dpi-desync-ttl=3 --dpi-desync-split-pos=1 --dpi-desync-repeats=20 --dpi-desync-fooling=md5sig,badseq --dpi-desync-fake-tls=/opt/etc/nfqws/tls_clienthello.bin"`
> 
> `# QUIC strategy NFQWS_ARGS_QUIC="--dpi-desync=fake --dpi-desync-repeats=20 --dpi-desync-cutoff=d4 --dpi-desync-fooling=badsum --dpi-desync-fake-quic=/opt/etc/nfqws/quic_initial.bin"`

тоже заработал ютб, спасибо. ростелеком

**Reply by SVY76** [reply]
Заработало на рт, спасибо!

**Reply by 233m** [reply]
> > `# HTTP(S) strategy NFQWS_ARGS="--filter-udp=50000-65535 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-cutoff=d4 --dpi-desync-repeats=20 --new --dpi-desync=fake,disorder2 --dpi-desync-ttl=3 --dpi-desync-split-pos=1 --dpi-desync-repeats=20 --dpi-desync-fooling=md5sig,badseq --dpi-desync-fake-tls=/opt/etc/nfqws/tls_clienthello.bin"`
> > `# QUIC strategy NFQWS_ARGS_QUIC="--dpi-desync=fake --dpi-desync-repeats=20 --dpi-desync-cutoff=d4 --dpi-desync-fooling=badsum --dpi-desync-fake-quic=/opt/etc/nfqws/quic_initial.bin"`
> 
> тоже заработал ютб, спасибо. ростелеком

спасибо добрый человек. заработало!

**Comment by jAba65** [comment]
Спасибо за конфиг, ютуб тоже стал работать

**Comment by x12lab** [comment]
> Тоже самое сделал, заработало:
> nfqws.conf:

Да, с таким yt заработал, спасибо. Пока только инсту не вышло восстановить, пока еще не работает (работала с другими настройками, но с ними не работал yt). Провайдер - проводной Билайн.

**Comment by filoisof** [comment]
Собственно, пришлось в кой-то веки завести аккаунт на гитхабе(мой первый пост), и все ради того, чтобы поблагодарить автора и всех кто помог и помогает в борьбе с темными силами:)
У меня работает все и на всех устройствах( тв, смарты, пк). Без затупливаний, без ожидания прогрузки и без тормозов. Конфиг стоковый заменил на тот, который выше.
Ну и от меня 5 копеек:
Чтобы твич заработал в узерлист добавляем:
```
assets.twitch.tv
twitch.tv
k.twitchcdn.net
cloudfront.net
d2v02itv0y9u9t.cloudfront.net
sb.scorecardresearch.com
```

**Reply by numbereleven-a** [reply]
> Собственно, пришлось в кой-то веки завести аккаунт на гитхабе(мой первый пост), и все ради того, чтобы поблагодарить автора и всех кто помог и помогает в борьбе с темными силами:) У меня работает все и на всех устройствах( тв, смарты, пк). Без затупливаний, без ожидания прогрузки и без тормозов. Конфиг стоковый заменил на тот, который выше. Ну и от меня 5 копеек: Чтобы твич заработал в узерлист добавляем:
> 
> ```
> assets.twitch.tv
> twitch.tv
> k.twitchcdn.net
> cloudfront.net
> d2v02itv0y9u9t.cloudfront.net
> sb.scorecardresearch.com
> ```

а давно твич в блоке?

**Reply by filoisof** [reply]
> а давно твич в блоке?

До вчерашнего дня никаких проблем у меня не было с твичем. Вчера YT окончательно перестал лечиться любыми версиями и настройками конфига GBDPI. После чего я установил nfqws и твич перестал грузить видео(и записи и трансляции) по аналогии с YT. Посмотрел в консоли ресурсы, добавил их в юзер лист, и твич заработал как прежде.

**Reply by numbereleven-a** [reply]
> > а давно твич в блоке?
> 
> До вчерашнего дня никаких проблем у меня не было с твичем. Вчера YT окончательно перестал лечиться любыми версиями и настройками конфига GBDPI. После чего я установил nfqws и твич перестал грузить видео(и записи и трансляции) по аналогии с YT. Посмотрел в консоли ресурсы, добавил их в юзер лист, и твич заработал как прежде.

не нормальная тема в целом, что то не так работает, но если устраивает дурение ненужного трафика то оставляй

**Comment by ZAMHome** [comment]
После обновления до версии 2.5.1 дополнительные правки nfqws.conf не понадобились, YouTube работает корректно с дефолтными настройками, прописанными в файл. Более того, с теми правками, что предлагаются выше, на версии 2.5.1 YouTube не работает.
Ultra (KN-1810), РТ (СПб).

**Reply by vchikalkin** [reply]
Подтверждаю, но есть нюанс:
В веб-версии и в Android приложении - работает превосходно
на Smart TV чуть подтормаживает перемотка и воспроизведение видео началось не сразу(долго крутился спиннер загрузки)

user.list чуть изменил:
ggpht.com
googleapis.com
googleusercontent.com
googlevideo.com
googleapis.com
gstatic.com
gvt1.com
i.ytimg.com
i9.ytimg.com
nhacmp3youtube.com
www.youtube.com
youtu.be
youtube-ui.l.google.com
youtube.com
youtubei.googleapis.com
yt3.ggpht.com
yt3.googleusercontent.com
yt4.ggpht.com
yt4.googleusercontent.com
ytimg.com
ytimg.l.google.com
ytstatic.l.google.com
youtube-nocookie.com
s.youtube.com

play.google.com
play.googleapis.com

secure.instagram.com
about.instagram.com
admin.instagram.com
preprod.instagram.com
payments.instagram.com
community.instagram.com
www.instagram.com
gateway.instagram.com
survey.instagram.com
pm.instagram.com
api.instagram.com
l.instagram.com
m.instagram.com
auth.instagram.com
maps.instagram.com
autodiscover.instagram.com
privacy.instagram.com
developers.instagram.com
help.instagram.com
parents.instagram.com
blog.instagram.com
i.instagram.com
business.instagram.com
upload.instagram.com
instagram.com
cdninstagram.com
facebook.com
fbcdn.net
facebook.net

**Comment by PavelAnok** [comment]
Ростелеком, Ярославль. v2.5.12, режим List - необходимы домены добавлены, QUIC отключен.
На стандартных настройках не работает. Поменял --dpi-desync-repeats=**20**, заработало, но работает не стабильно. Главная страница открывается мгновенно, поиск работает быстро, превью показывает, но как запускаешь видео, то нужно ждать секунд 20. После запуска видео все работает штатно. При попытке перемотать случается задержка на 10-20 секунд или даже вообще может черный экран и видео не воспроизводится до перезапуска. Шортсы воспроизводятся с перебоями, штуки три работаю, потом зависает, перелистываешь туда-сюда, опять начинают работать.
Может есть у кого настройки для победы над Ростелекомом в Ярославле?

**Reply by elijah0107** [reply]
Такая же проблема. Часто воспроизводится именно по вечерам. Днем и утром всё грузит без проблем

**Comment by EasyKill33** [comment]
Все работает идеально: ютуб, дискорд, трекеры и т.п., но сайт instagram.com через браузер не открывает ни с телефона(через браузер), ни с компа. Но если заходить через приложение инстаграма на телефоне то все идеально работает. Домены инсты добавлены в юзер лист. Может кто знает возможно ли чтобы инста открывалась с пк?

**Reply by vchikalkin** [reply]
Попробуйте этот список:

secure.instagram.com
about.instagram.com
admin.instagram.com
preprod.instagram.com
payments.instagram.com
community.instagram.com
www.instagram.com
gateway.instagram.com
survey.instagram.com
pm.instagram.com
api.instagram.com
l.instagram.com
m.instagram.com
auth.instagram.com
maps.instagram.com
autodiscover.instagram.com
privacy.instagram.com
developers.instagram.com
help.instagram.com
parents.instagram.com
blog.instagram.com
i.instagram.com
business.instagram.com
upload.instagram.com
instagram.com
cdninstagram.com
facebook.com
fbcdn.net
facebook.net

cdn.fbsbx.com

**Reply by EasyKill33** [reply]
Список такой же, единственное добавил самый последний домен, но это не помогло(

**Reply by vchikalkin** [reply]
В таком случае стоит копать в сторону конфига.
И DNS использовать не от провайдера с TLS
Какой у вас провайдер?

**Reply by EasyKill33** [reply]
DNS использую платный от adguard. Провайдер местный городской, трафик вроде берет у МТС.

**Reply by PavelAnok** [reply]
> Попробуйте этот список:
> 
> secure.instagram.com about.instagram.com admin.instagram.com preprod.instagram.com payments.instagram.com community.instagram.com [www.instagram.com](http://www.instagram.com) gateway.instagram.com survey.instagram.com pm.instagram.com api.instagram.com l.instagram.com m.instagram.com auth.instagram.com maps.instagram.com autodiscover.instagram.com privacy.instagram.com developers.instagram.com help.instagram.com parents.instagram.com blog.instagram.com i.instagram.com business.instagram.com upload.instagram.com instagram.com cdninstagram.com facebook.com fbcdn.net facebook.net
> 
> cdn.fbsbx.com

Я раньше тоже указывал поддомены, но в этом нет смысла, достаточно указать основной домен и все будет работать, проверено

**Reply by vchikalkin** [reply]
> > Попробуйте этот список:
> > 
> > secure.instagram.com about.instagram.com admin.instagram.com preprod.instagram.com payments.instagram.com community.instagram.com [www.instagram.com](http://www.instagram.com) gateway.instagram.com survey.instagram.com pm.instagram.com api.instagram.com l.instagram.com m.instagram.com auth.instagram.com maps.instagram.com autodiscover.instagram.com privacy.instagram.com developers.instagram.com help.instagram.com parents.instagram.com blog.instagram.com i.instagram.com business.instagram.com upload.instagram.com instagram.com cdninstagram.com facebook.com fbcdn.net facebook.net
> > 
> > cdn.fbsbx.com
> 
> Я раньше тоже указывал поддомены, но в этом нет смысла, достаточно указать основной домен и все будет работать, проверено

Странно
Попробовал без поддоменов - перестало работать

**Reply by PavelAnok** [reply]
Странно, я проверил на всех своих устройствах и на каждом все работает отлично. В Авто лист ничего не добавляется, значит указания основных доменов достаточно и все открывается.

**Comment by AlexDiag** [comment]
Люди добрый, помогите пожалуйста. Потратил полторы недели уже, не могу решить задачку. Роутер Keenetic, провайдер Beeline (4G), стоит zapret на роутере. Подобрал параметры настройки, компьютеры работают отлично в браузере Chrome, скорость связи во время воспроизведения видео от 20 до 50 Мбит. Телефон Android тоже по WiFi работает, подкачка буфера короткими импульсами раз в несколько секунд на скорости от 7-8 до 40 Мбит. И не работает телевизор, Sony модели 22 года на Android 10, подключен по WiFi как и телефон. На телевизоре штатное приложение Ютуба, новейшая версия. Оболочка запускается отлично, поиск отлично работает. Воспроизведение видео запускается или совсем без задержки или с задержкой до 5 сек. И далее начинается интересное. Скорость связи очень сильно изменяется. Может до десятка минут быть как на компьютере, несколько десятков Мбит. Но потом скорость начинает падать, вначале до единиц Мбит в течении пары минут. Потом до 600-900 Кбит/сек. Начинается уменьшение буфера воспроизведения. Потом скорость падает до 150-350 Кбит/сек, обвал буфера. Может сразу при запуске воспроизведения начинать с очень малых скоростей и почти сразу обвал буфера. Очень различно себя ведет, но в конечном итоге при разных сценариях изменения скорости приходит все равно к одному- скорость дико падает и обвал буфера.
 Ничего подобного на телефоне, работающем по этому же WiFi не наблюдается. Как и на компьютере, скорость почти постоянна.
 Обход блокировки Ютуб причем у ТВ явно работает, потому что если выключить zapret то будет 150-200 кб/сек всегда и более 5-10 сек смотреть невозможно. 
 ТВ не работает по UDP и не работает по протоколу Квик, что касается воспроизведения самого видео. Это проверялось. Видео идет у ТВ по TLS 1.3. Квик влияет на загрузку оболочки Ютуба, и в zapret выключен из обработки вообще - так оболочка загружается лучше всего.
 Как добиться нормальной работы ТВ я уже совсем не понимаю. Чего только не делал. TTL менял в настройках- выше определенного значения не влияет, а обвал буфера идет с любым значением. Значение repeat менял от 6 до 20, никак поведение видео на ТВ не изменилось. 
 Почему может так сильно плавать скорость связи при воспроизведении видео и почему-то только у ТВ ? Что вообще делать в таких ситуациях ?

**Reply by PavelAnok** [reply]
Проверьте, что отключен трафик UDP на порт 443 (иначе видео будет тормозить на телевизоре) ,и попробуйте на телевизоре прописать dns 9.9.9.9. Если первый пунк поможет, второй делать не обязательно.
![UDP](https://github.com/user-attachments/assets/edfc69d6-0f3c-46cc-a4bd-4110c19ad260)

**Comment by AlexDiag** [comment]
Пробовал и закрыть 443 порт по UDP, и различные DNS назначать как серверы DNS в роутере, ничего не меняется. DNS провайдера отключены, но и когда включены были картина на ТВ была точно такой же. Если же закрыть порт 80 UPD, как у Вас на фото- оболочка Ютуба на ТВ не загружается

**Reply by orakult** [reply]
Присоединяюсь.
Переделал все выше перечисленное, ничего не помогает на GoogleTV.

**Reply by AlexDiag** [reply]
А я почти победил проблемы штатного Youtube на Android TB. Подсказка- добейтесь работы воспроизведения в браузере по h3 (Quick). Чтобы устойчиво с h3 работало в браузере. И разделите стратегии по Ютубу и всему остальному. Для того чтобы только для youtube.list назначить autottl. 
Пока стопроцентную инструкцию не могу написать, тестирую пока. Но результаты обнадеживают вроде бы.
